mmv1/products/composer/UserWorkloadsConfigMap.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'UserWorkloadsConfigMap'
+description: |
+  User workloads ConfigMap used by Airflow tasks that run with Kubernetes Executor or KubernetesPodOperator.
+  Intended for Composer 3 Environments.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  # TODO: add v1 reference when this is moved to ga
+  api: https://cloud.google.com/composer/docs/reference/rest/v1beta1/projects.locations.environments.userWorkloadsConfigMaps
+
+min_version: beta
+
+base_url: projects/{{project}}/locations/{{region}}/environments/{{environment}}/userWorkloadsConfigMaps
+self_link: projects/{{project}}/locations/{{region}}/environments/{{environment}}/userWorkloadsConfigMaps/{{name}}
+
+# Overrides one or more timeouts, in minutes. All timeouts default to 20.
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 1
+  update_minutes: 1
+  delete_minutes: 1
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "composer_user_workloads_config_map_basic"
+    primary_resource_id: "config_map"
+    vars:
+      environment_name: "test-environment"
+      config_map_name: "test-config-map"
+
+parameters:
```

Comment:
There are `parameters` and `properties` sections of the YAML for storing information about user-supplied values. In general a resource's [fields should be nested under `properties`](https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-fields) and only added to `parameters` when needed (i.e if url_param_only=true).

Please move all the fields that aren't url_param_only=true under a top-level `properties` field in the YAML.

Reply:
Also, could you also please add a new field `project` as a url_param_only=true parameter? Currently the code generator is identifying that a project value is needed in the URLs for managing the resource and, as a result, is adding a project field to the resource automatically. It'd be good to make the field explicitly defined in the YAML instead.

Reply:
When I defined project field, it was still generated and I got double definition in the final code. I'm getting this error as a result (on build) : 
```
google-beta/services/composer/resource_composer_user_workloads_config_map.go:91:4: duplicate key "project" in map literal
```